### PR TITLE
Windows Build

### PR DESCRIPTION
**Pull request by JcBernack**
**Note**: #1 and #2 need to be merged first

- fixed missing symbols in Windows build
- fixed memory management issues
- fixed usage of handles in `libevm`
  - fixed [EVM-762](https://horizenlabs.atlassian.net/browse/EVM-762)
  - fixed concurrency panic
- added a simple libevm load test

### CODE REVIEW

File: libevm/lib/handles.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		mutex:   &sync.RWMutex{},
+	}
+}
+
+func (h *Handles[T]) Add(obj T) int {
+	h.mutex.Lock()
+	defer h.mutex.Unlock()
+	// pedantic safety check for overflow: this is highly unlikely because there are 2**31-1 available handles,
+	// just the memory consumption alone will likely cause problems before this happens
+	if len(h.used) == math.MaxInt32 {
```
Comment (i-Alex) [inline libevm/lib/handles.go:31]: i'm not sure about this check.
From the code below:
```
if h.current == math.MaxInt32 {
		h.current = 0
}
h.current++
```
I see, that it is possible to have `current == MaxInt32` -> that will lead to `len(h.used)` to be exactly `math.MaxInt32`

I think, we need to change the code below to `if h.current == math.MaxInt32 - 1`
or something like this

Comment (JcBernack) [reply libevm/lib/handles.go:31]: > I see, that it is possible to have current == MaxInt32 -> that will lead to len(h.used) to be exactly math.MaxInt32

Correct, in that situation the next item doesnt fit anymore, which is why the check is for `len(h.used) == math.MaxInt32`. I think it is ok like this. The system will most likely be out of memory at this point anyway.

Comment (i-Alex) [reply libevm/lib/handles.go:31]: hes sense!
thanks

**Review by i-Alex — COMMENTED** [review]
Looks fine.
The only minor non-code issue I see is about the `.run` folder path.
I'm not sure how do you open the project, but in my case:
1) If I open it in a repo root, then all run configs pathes are broken
2) If I open it in `evm` -> run `.run` is not seen. Folder must be moved inside `evm`
Both ways are fine to me, but let's adopt the configs.
We may discuss if needed

**Review by i-Alex — APPROVED** [review]

**Pull request merged**